WordPress/gutenberg
# Library: Update icons in the creation menu

## What?
Update the icons associated with the "Create template part" and "Create pattern" actions.

## Why?
See https://github.com/WordPress/gutenberg/issues/52043

## Testing Instructions
* Navigate to Site Editor -> Library
* Click +
* Observe updated icons

### Before
<img width="239" alt="Screenshot 2023-06-29 at 11 01 01" src="https://github.com/WordPress/gutenberg/assets/846565/88925b85-5c62-4218-bcc2-258b75130062">

### After
<img width="245" alt="Screenshot 2023-06-29 at 11 01 12" src="https://github.com/WordPress/gutenberg/assets/846565/ed83f771-2bb0-4ceb-9323-4b7cf73f6283">
